Resources/Private/Styles/Main.scss

```diff
@@ -12,3 +12,4 @@
 @import "Elements/RightBar";
 @import "Elements/LeftBar";
 @import "Elements/Other";
+@import "Elements/Ckeeditor";
```

Comment:
Rename the file to Resources/Private/Styles/Elements/_CkeEditor.scss for convention.